File: LeetCode-Programs/Kids With the Greatest Number of Candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        vector<bool> vec;
        //sort(candies.begin(), candies.end());
        int max=*max_element(candies.begin(), candies.end());
        for(int i=0;i<candies.size();i++)
        {
           if(candies[i]+extraCandies>=max)
               vec.push_back(true);
            else
                vec.push_back(false);
        }
        return vec;
    }
};

File: LeetCode-Programs/Kids With the Greatest Number of Candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        new_list=[]
        maxx=max(candies)
        for i in candies:
            if i+extraCandies>=maxx:
                new_list.append(True)
            else:
                new_list.append(False)
        return new_list
